src/Service/PagePurger.php

```diff
@@ -43,6 +44,24 @@ public function purge( Page $page ) {
 	}
 
 	/**
+	 * @since 0.x
+	 *
+	 * @param Pages $pages
+	 *
+	 * @return bool
+	 *
+	 */
+	public function purgePages( Pages $pages ) {
+		$pagesArray = $pages->toArray();
+
+		foreach ( $pagesArray as $page ) {
+			$this->purge( $page );
+		}
+
+		return true;
```

Comment:
What's the point of always returning true? I'd say either leave this out (and update the docblock to `@return void`) or have explain when this will return true and when other things might happen.

I realise that this is just following the pattern set by the `purge()` method, but I'd question that one too; perhaps we can make them both better?